carrot-core/src/main/java/org/bigbase/carrot/BigSortedMap.java

```diff
@@ -2281,13 +2282,12 @@ public void snapshot() {
       // Save store meta data
       saveStoreMeta(fc);
     } catch (IOException e) {
-      System.err.println(
-          "Snapshot failed. Can not create snapshot file: " + snapshotFile.getAbsolutePath());
-      e.printStackTrace();
+      log.error("Snapshot failed. Can not create snapshot file: " + snapshotFile.getAbsolutePath());
```

Comment:
Still concatenation of strings

Reply:
Ups. Yes. Idea set me up...
In progress...

Reply:
done